rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -556,7 +573,7 @@ void np_handler::operator()()
 			case NotificationType::UserJoinedRoom: notif_user_joined_room(notif.second); break;
 			case NotificationType::UserLeftRoom: notif_user_left_room(notif.second); break;
 			case NotificationType::RoomDestroyed: notif_room_destroyed(notif.second); break;
-			case NotificationType::SignalP2PEstablished: notif_p2p_established(notif.second); break;
+			case NotificationType::SignalP2PConnect: notif_p2p_connect(notif.second); break;
 			case NotificationType::RoomMessageReceived: notif_room_message_received(notif.second); break;
```

Comment:
Is this missing the '_SignalP2PDisconnect' notification?
Even if it is just an empty break..

Reply:
It's not implemented on server so no point atm(_ is a prefix for unused items in rust).